api/src/main/java/javax/enterprise/inject/spi/Decorator.java

```diff
@@ -25,7 +25,10 @@
  * <p>
  * Represents an enabled {@linkplain javax.decorator decorator}.
  * </p>
- * 
+ * <p>
+ * Since CDI 2.0, an implementation this interface may implement {@link Prioritized} to be enabled for the entire application with a priority value.
```

Comment:
This sentence does not really make sense. I suggest following changes:
1) `an implementation this interface` -> `an implementation of this interface`
2) `to be enabled for the entire application with a priority value.` -> `in order to enable the decorator with given priority value for entire application`

Plus the same changes for `api/src/main/java/javax/enterprise/inject/spi/Interceptor.java `